source/dub/dub.d

```diff
@@ -285,11 +307,48 @@ class Dub {
 			}
 		}
 
+		// Override user + local package path from /etc/dub/settings.json
+		// Then continues loading local settings from these folders. (keeping
+		// global /etc/dub/settings.json settings intact)
+		//
+		// Don't use it if either $DPATH or $DUB_HOME are set.
+		if (!overrideUserSettings) {
+			if (m_config.userSettings.length) {
+				m_dirs.userSettings = NativePath(m_config.userSettings.expandEnvironmentVariables);
```

Comment:
What happens if the environment variable can't get expanded ?

Reply:
nothing is inserted in that case, (variable replaced with empty string) probably worth putting into dub-docs

Reply:
nothing is inserted in that case, (variable replaced with empty string) probably worth putting into dub-docs

Reply:
I think it should error out.

Reply:
done